packages/sei-cosmwasm/src/sei_types.rs

```diff
@@ -68,41 +65,33 @@ pub struct Order {
     pub nominal: Decimal, // only needed for Fokmarketbyvalue order
 }
 
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+#[serde(rename_all = "snake_case")]
+pub struct Cancellation {
+    pub id: u64,
+    pub contract_address: String,
+    pub price_denom: String,
+    pub asset_denom: String,
+    pub order_type: OrderType,
+    pub position_direction: PositionDirection,
+    pub price: Decimal,
```

Comment:
what is the purpose of having all of this information for a cancellation?

Reply:
This is just matching the sei-chain type (other than Initiator which isn't used)